client/app/veeam/dashboard/veeam-dashboard.html

```diff
@@ -1,58 +1,123 @@
+                    </oui-tile-definition>
+                </div>
+            </oui-tile>
         </div>
         <div class="col-xm-12 col-md-4">
-            <cui-tile-subscription data-loading="VeeamDashboardCtrl.subscriptionInfos.loading"
-                data-subscription-info="VeeamDashboardCtrl.subscriptionInfos.data"
-                data-offer-actions="VeeamDashboardCtrl.uiActions.changeOffer"
-                data-autorenew-actions="VeeamDashboardCtrl.uiActions.manageAutorenew"
-                data-contact-actions="VeeamDashboardCtrl.uiActions.manageContact"
-                class="h-100"></cui-tile-subscription>
+            <oui-tile 
+                data-heading="{{ ::'common_tile_subscription' | translate }}"
+                data-loading="$ctrl.subscriptionInfos.loading"
+                class="oui-tile-h-100 h-100">
+                <oui-tile-definition data-ng-if="$ctrl.subscriptionInfos.loading">
+                    <oui-tile-description>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                        <oui-skeleton data-randomized></oui-skeleton>
+                    </oui-tile-description>
+                </oui-tile-definition>
+                <div data-ng-if="!$ctrl.subscriptionInfos.loading">
+                    <oui-tile-definition
+                        data-ng-if="$ctrl.subscriptionInfos.data.offer" 
+                        data-term="{{ ::'common_tile_subscription_offer' | translate }}" data-description="{{ ::$ctrl.subscriptionInfos.data.offer }}">
+                        <oui-action-menu data-compact data-align="end" data-ng-if="$ctrl.uiActions.changeOffer.isAvailable()">
+                            <oui-action-menu-item data-text="{{::$ctrl.uiActions.changeOffer.text}}"
+                                data-aria-label="{{::$ctrl.uiActions.changeOffer.text}}"
+                                data-on-click="$ctrl.uiActions.changeOffer.callback()">
+                            </oui-action-menu-item>
+                        </oui-action-menu>
+                    </oui-tile-definition>
+                    <oui-tile-definition
+                        data-term="{{($ctrl.subscriptionInfos.data.renew.forced || $ctrl.subscriptionInfos.data.renew.automatic) ? 'common_tile_subscription_autorenew' : 'common_tile_subscription_expiration_date' | translate}}"
+                        data-description="{{ ::$ctrl.subscriptionInfos.data.expiration | momentFormat:'LL' }}">
+                        <oui-action-menu data-compact data-align="end" data-ng-if="$ctrl.uiActions.manageAutorenew.isAvailable()">
```

Comment:
Alignment of html attributes ? (For example) :
`<oui-action-menu data-compact data-align="end" 
                              data-ng-if="$ctrl.uiActions.manageAutorenew.isAvailable()">`
Or in a way you prefer.